src/main/java/pl/bajerska/befindyourmeal/recipe/RecipeController.java

```diff
@@ -0,0 +1,32 @@
+package pl.bajerska.befindyourmeal.recipe;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import pl.bajerska.befindyourmeal.edamam.ApiService;
+
+@Controller
+public class RecipeController {
+
+    private ApiService apiService;
+
+    public RecipeController(ApiService apiService) {
+        this.apiService = apiService;
+    }
+
+    @RequestMapping(value = {"describerecipe"}, method = RequestMethod.GET)
+    public String getRecipe(Model model) {
+        RecipeCriteria recipeCriteria = new RecipeCriteria();
+        recipeCriteria.ingredients.add("");
```

Comment:
You shouldn't be able* and shouldn't need** to do that! 
*) please check comment in RecipeCriteria
**) if it need to be initialized that way, it can be done by RecipeCriteria constructor.

The less RecipeController knows about the internal details of RecipeCriteria, the better. :)